supportApp/GraphicsMagickSrc/jp2/src/libjasper/jpc/jpc_tsfb.c

```diff
@@ -82,10 +82,10 @@
 #include "jpc_util.h"
 #include "jpc_math.h"
 
-int jpc_tsfb_analyze2(jpc_tsfb_t *tsfb, int *a, int xstart, int ystart,
-  int width, int height, int stride, int numlvls);
+int jpc_tsfb_analyze2(jpc_tsfb_t *tsfb, jpc_fix_t *a, int xstart, int ystart,
+  unsigned width, unsigned height, unsigned stride, unsigned numlvls);
```

Comment:
This change is not explained in the commit message.

PS: I see it gets later reverted in 2483643d2e5c705e63ecae650d4c42fce2ec7232. I'd squash all next patches into this one. We don't need to go back and forth with these changes. It just makes the review harder.